dlproject/data/renoir_dataset.py

```diff
@@ -7,14 +7,14 @@
 
 
 class RENOIRDatasetSequence(tf.keras.utils.Sequence):
-    def __init__(self, dataset_folder, dataset_type, batch_size, target_size=(5328, 3000)):
+    def __init__(self, dataset_folder, dataset_type, batch_size, patch_size=(256, 256)):
         if dataset_type not in ['Mi3_Aligned', 'T3i_Aligned']:
             raise ValueError("Invalid dataset type. Choose 'Mi3_Aligned' or 'T3i_Aligned'.")
 
         self.dataset_folder = dataset_folder
         self.dataset_type = dataset_type
         self.batch_size = batch_size
-        self.target_size = target_size
+        self.patch_size = patch_size
         self.test_data_paths = self.__get_test_data_paths()
 
     def __len__(self):
```

Comment:
Con questa modifica però `len(renoir_sequence)` non ritorna più il valore corretto giusto? Perché per ciascuna immagine ne vengono generate altre. Per questo mi pareva più semplice ridimensionare le immagini.

Reply:
Pensavo di aver eliminato la classe RENOIRDatasetSequence... non serve più perché dal notebook Restormer.ipynb possiamo caricare direttamente una cartella con le immagini (sezione "3. Upload Images")

Reply:
Ah ok non aveva visto scusa, allora approvo.

Reply:
L'idea è la seguente:
- si scaricano il immagini del dataset RENOIR
- si chiama la funzione `preprocess_renoir_dataset.py` indicando come parametro la cartella di destinazione in cui salvare le patches
- si caricano le patches tramite il notebook `Restormer.ipynb`

Siccome alla fine abbiamo un totale di 128 immagini (se si sceglie la dimensione dei patch 1024x1024) mi sembrava più semplice caricarle direttamente così